src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/understand-technical-concepts/alert-title-template.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Title template for NRQL Alert Conditions
+tags:
+  - Alerts and applied intelligence
+  - Alerts
+  - Alert conditions
+metaDescription: 'Give your alert condition a unique title to easily identify related incidents.'
+redirects:
+  - /docs/alerts/new-relic-alerts/defining-conditions/alert-condition-descriptions
+  - /docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-descriptions
+  - /docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-custom-violation-descriptions
+  - /docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/advanced-techniques/alert-custom-violation-descriptions/
+freshnessValidatedDate: never
+---
+
+import alertsAddingAlertDetailstoanAlertCondition from 'images/alerts_screenshot-crop_adding-alert-details-to-an-alert-condition.webp'
+
+
+A title template is used when incidents are opened by the condition. It overrides the default title.
+
+## Why use a title template?
+An alert condition is the foundation for incident creation. Assigning a meaningful title to your alert condition ensures you can quickly pinpoint the source and specific parameters that triggered an incident. Whether you're using our guided mode or crafting a condition from a chart or policy, leverage our title template in the final step to customize incident titles for easy identification.
```

Comment:
Can you change `Assigning a meaningful title to your alert condition` to `Assigning a meaningful title template to your alert condition`

Reply:
Yes! Will do.